app/services/proofing/aamva/request/verification_request.rb

```diff
@@ -10,27 +10,53 @@
 module Proofing
   module Aamva
     module Request
+      RequestAttribute = Data.define(:xpath, :required).freeze
       class VerificationRequest
         CONTENT_TYPE = 'application/soap+xml;charset=UTF-8'
         DEFAULT_VERIFICATION_URL =
           'https://verificationservices-cert.aamva.org:18449/dldv/2.1/online'
         SOAP_ACTION =
           '"http://aamva.org/dldv/wsdl/2.1/IDLDVService21/VerifyDriverLicenseData"'
 
+        VERIFICATION_REQUESTED_ATTRS = {
+          first_name: RequestAttribute.new('//nc:PersonGivenName', true),
+          middle_name: RequestAttribute.new('//nc:PersonMiddleName', false),
```

Comment:
totally optional but if we wanted to trade some terseness for readability, we could use the keywords:

```suggestion
          first_name: RequestAttribute.new(xpath: '//nc:PersonGivenName', required: true),
          middle_name: RequestAttribute.new(xpath: '//nc:PersonMiddleName', required: false),
```